doc/api/fs.md

```diff
@@ -2086,6 +2086,16 @@ fs.mkdir('/tmp/a/apple', { recursive: true }, (err) => {
 });
 ```
 
+`fs.mkdir` has non-enumerable non-writable property `Symbol('recursive')` that
+can be used to detect if the recursive feature is available
+
+```js
+Object.getOwnPropertySymbols(fs.mkdir).map((sym) => {
```

Comment:
What if instead of a symbol specific to this feature we exposed a "feature" symbol? It could be reused in other methods, and would be well known, so it wouldn't require looping over properties.

Reply:
Sounds interesting! Are you imagining that `Symbol('features')` would then have an array with a list of all features? Can you share a code example of how userland feature detection would look?

Reply:
I was thinking that `Symbol('features')` would be public API, which is how users would know to access it. Then, I'd do something like:

```js
fs.mkdir[kFeaturesSymbol] = { recursive: true };
```

Open to suggestions on that implementation, but I'd go with an object over an array because it's a bit more flexible to work with IMO.

From a user's perspective:

```js
const features = `require('util').FEATURES_SYMBOL;` // also open to suggestions here.
const { mkdir } = require('fs');

if (typeof features == 'symbol' && mkdir[features].recursive) {
  // use the new thing
} else {
  // do the old thing
}
```

Reply:
Just pushed a commit to take this approach, lmk what you think